api/http/handler/edgestacks/edgestack_inpect_config.go

```diff
@@ -0,0 +1,60 @@
+package edgestacks
+
+import (
+	"net/http"
+	"path"
+
+	httperror "github.com/portainer/libhttp/error"
+	"github.com/portainer/libhttp/request"
+	"github.com/portainer/libhttp/response"
+	portainer "github.com/portainer/portainer/api"
+)
+
+type (
+	configResponse struct {
+		Prune            bool
+		StackFileContent string
+	}
+)
+
+// GET request on /api/edge_stacks/:id/config?endpointId
```

Comment:
The fact that we connect endpoint and stacks under /api/edge_stacks bothers me.

As well as having multiple Edge entrypoint (Edge only operation, query by the agent) under different resources

I think that everything that is Edge agent exclusive API should be under `/api/endpoints/:id/edge/` since an Edge agent is tied to an endpoint.

What about relocating this endpoint to `/api/endpoints/:id/edge/stacks/:stackId`.

Reply:
done